src/toil/test/src/fileStoreTest.py

```diff
@@ -261,8 +261,7 @@ def _testWriteReadGlobalFilePermissions(job, executable):
 
             for mutable in True,False:
                 for symlink in True,False:
-                    #with self.subTest(f'Now testing readGlobalFileWith: mutable={mutable} symlink={symlink}'):
-                    dstFile = job.fileStore.getLocalTempFile() + str(uuid4())
+                    dstFile = job.fileStore.getLocalTempDir() + str(uuid4())
```

Comment:
This might need to be an `os.path.join()` call instead of a `+`? Or does `getLocalTempDir()` always returns a path ending in `/`?